src/os/bluestore/aio.h

```diff
@@ -16,12 +16,12 @@ struct aio_t {
   int fd;
   boost::container::small_vector<iovec,4> iov;
   uint64_t offset, length;
-  int rval;
+  uint64_t rval;
   bufferlist bl;  ///< write payload (so that it remains stable for duration)
 
   boost::intrusive::list_member_hook<> queue_item;
 
-  aio_t(void *p, int f) : priv(p), fd(f), offset(0), length(0), rval(-1000) {
+  aio_t(void *p, int f) : priv(p), fd(f), offset(0), length(0), rval(0) {
```

Comment:
i'd suggest leave as it is. -1000 represent a value not yet initialized, and it is not equal to any errno.